libs/auth/scripts/configure_java_card.ts

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable no-console */
+import { Buffer } from 'buffer';
+import { spawnSync } from 'child_process';
+import { assert, sleep } from '@votingworks/basics';
+import { Byte } from '@votingworks/types';
+
+import { CommandApdu, constructTlv } from '../src/apdu';
+import {
+  CARD_VX_ADMIN_CERT,
+  CARD_VX_CERT,
+  DEFAULT_PIN,
+  GENERIC_STORAGE_SPACE,
+  JavaCard,
+  MAX_INVALID_PIN_ATTEMPTS,
+  OPEN_FIPS_201_AID,
+  PUK,
+  VX_ADMIN_CERT_AUTHORITY_CERT,
+} from '../src/java_card';
+import {
+  construct8BytePinBuffer,
+  CRYPTOGRAPHIC_ALGORITHM_IDENTIFIER,
+} from '../src/piv';
+
+const APPLET_PATH = 'applets/OpenFIPS201-v1.10.2-with-vx-mods.cap';
+const GLOBAL_PLATFORM_PATH = 'scripts/gp.jar';
+
+/**
+ * CHANGE REFERENCE DATA ADMIN is an OpenFIPS201-specific extension of the PIV-standard CHANGE
+ * REFERENCE DATA command. It allows setting PINs and PUKs without the current value. APDUs must be
+ * sent over a GlobalPlatform Secure Channel.
+ */
+const CHANGE_REFERENCE_DATA_ADMIN = {
+  INS: 0x24,
+  P1: 0xff,
+  P2_PIN: 0x80,
+  P2_PUK: 0x81,
+} as const;
+
+/**
+ * PUT DATA ADMIN is an OpenFIPS201-specific extension of the PIV-standard PUT DATA command, meant
+ * for initial card configuration. APDUs must be sent over a GlobalPlatform Secure Channel.
+ */
+const PUT_DATA_ADMIN = {
+  INS: 0xdb,
+  P1: 0x3f,
+  P2: 0x00,
+
+  UPDATE_CONFIG_TAG: 0x68,
+  PIN_POLICY_TAG: 0xa0,
+  MAX_INVALID_PIN_ATTEMPTS_OVER_CONTACT_INTERFACE_TAG: 0x86,
+
+  CREATE_KEY_TAG: 0x66,
+  CREATE_OBJECT_TAG: 0x64,
+  ID_TAG: 0x8b,
+  ACCESS_MODE_OVER_CONTACT_INTERFACE_TAG: 0x8c,
+  ACCESS_MODE_OVER_CONTACTLESS_INTERFACE_TAG: 0x8d,
+  ACCESS_MODE_NEVER: 0x00,
+  ACCESS_MODE_PIN_GATED: 0x12,
+  ACCESS_MODE_ALWAYS: 0x7f,
+  KEY_MECHANISM_TAG: 0x8e,
+  KEY_ROLE_TAG: 0x8f,
+  KEY_ROLE_SIGN: 0x04,
+  KEY_ATTRIBUTE_TAG: 0x90,
+  KEY_ATTRIBUTE_NONE: 0x00,
+} as const;
+
+function getEnvVar(envVarName: string, required = true): string {
+  const value = process.env[envVarName];
+  if (required) {
+    assert(value !== undefined, `Missing required ${envVarName} env var`);
+  }
+  return value || '';
+}
+
+const vxCertAuthorityCertPath = getEnvVar('VX_CERT_AUTHORITY_CERT_PATH');
+const vxOpensslConfigPath = getEnvVar('VX_OPENSSL_CONFIG_PATH');
+const vxPrivateKeyPassword = getEnvVar('VX_PRIVATE_KEY_PASSWORD');
+const vxPrivateKeyPath = getEnvVar('VX_PRIVATE_KEY_PATH');
```

Comment:
As noted in the PR description, the env vars that this script requires will almost certainly change once I tackle #3055 (using a TPM-held private key on real hardware).